<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>370</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>qtcsdr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>621</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleAM">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>AM</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Offset:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinFreq">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1999999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>89500000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinOffset">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1200000</number>
     </property>
     <property name="maximum">
      <number>1200000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>200000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleWFM">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>WFM</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleUSB">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>USB</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleLSB">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>LSB</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleRun">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>52</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleNFM">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>NFM</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QMySpectrumWidget" name="widgetFFT" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>601</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMySpectrumWidget</class>
   <extends>QWidget</extends>
   <header>qmyspectrumwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
